pkg/rpc/response/result/tx_raw_output.go

```diff
@@ -15,7 +15,7 @@ import (
 // a Transaction.
 type TransactionOutputRaw struct {
 	transaction.Transaction
-	TransactionMetadata
+	*TransactionMetadata
```

Comment:
But doesn't `omitempty` solve this problem?

Reply:
Not sure, because we have to keep both `Transaction` and `TransactionMetadata` at the same level. So there's an additional logic after `Transaction` and `TransactionMetadata` marshalling where we still have to check whether `TransactionMetadata` exists or not.

Reply:
I thought that `TransactionMetadata` with default values (and `omitempty` for every field) will be marshaled into `{}` which can be concatenated with `Transaction` easily?

Reply:
Yes, this solves the problem, fixed.